repository/userRepository.py

```diff
@@ -0,0 +1,159 @@
+# w plikach typu repository, przechowuje sie metody i zapytania czysto odpowiedzialne za CRUD zapytan bazodanowych
+
+import sqlite3
+
+
+# todo we will be working on database during all operations, there
+# should be open connection in main class and somehow repos and everything should operate on this to
+# avoid opening and closing this everytime
+
+def initiate_db():
+    conn = sqlite3.connect('database/gitfetcherdb.db')
```

Comment:
zrob dodatkowy folder  o nazwie `utils` i w srodku plik z nazwa `constans.py` i tam zmienna `DATABASE = 'database/gitfetcherdb.db'` i bedziemy to wszedzie importowac zeby sie nie powtarzac.